pocs/tests/test_observatory.py

```diff
@@ -69,20 +87,44 @@ def test_bad_mount_driver(config):
         Observatory(simulator=simulator, config=conf, ignore_local_config=True)
 
 
-def test_bad_scheduler(config):
+def test_can_observe_no_scheduler(config):
     conf = config.copy()
-    simulator = ['all']
-    conf['scheduler']['type'] = 'foobar'
-    with pytest.raises(error.NotFound):
-        Observatory(simulator=simulator, config=conf, ignore_local_config=True)
+    cameras = create_cameras_from_config(conf)
```

Comment:
No reason to add the cameras to this test. The Observatory should be able to be created without cameras and you are trying to test the scheduler here, so you want to make a minimal example required to test the scheduler.

Reply:
Also, you've removed the test for a badly named scheduler, which is different from no scheduler.

Reply:
I guess you have that down in the test_scehduler, so that is okay.

Reply:
You can capture the logs that are output and make sure it is outputting the correct log message rather than just checking for the False below.